ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,41 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid"),
+                                           shape=helpers.ints(min_value=0).
+                                           map(lambda x: tuple([x, ]))),
+    dim=helpers.ints(min_value=-1, max_value=0),

```

Comment:
The generation of the axis should depend on the input array's shape and size, otherwise the axis might be invalid. You could use `dtype_values_axis` to get correct array-axis pairs.